docs/how-to-guides/block-tutorial/writing-your-first-block-type.md

```diff
@@ -1,137 +1,180 @@
-# Writing Your First Block Type
+# Create a basic block
 
-To keep things simple for our first example, let's create a new block type which displays a styled message in a post. At this point, we won't allow the user to edit the message. We'll learn more about editable fields in later sections.
+This guide takes you through creating a basic block to display a message in a post. This message will be fixed, we won't allow the user to edit the message, the goal of the guide is to show how to register and load a block.
 
-Blocks containing static content are implemented entirely in JavaScript using the `registerBlockType` function. This function is responsible for specifying the blueprint of a block, describing the behaviors necessary for the editor to understand how it appears, changes when edited, and is ultimately saved in the post's content.
+## Overview
 
-## Enqueuing Block Scripts
+There are two main types of blocks: static and dynamic, this guide focusing on static blocks. A static block is used to insert HTML content into the post and saved with the post. A dynamic block builds the content on the fly when rendered on the front end, see the [dynamic blocks guide](/docs/how-to-guides/block-tutorial/creating-dynamic-blocks.md).
```

Comment:
I had some trouble processing this section in the first go, would it help to apply some small tweaks like:
```suggestion
There are two main types of blocks: static and dynamic, this guide focuses on static blocks. A static block is used to insert HTML content into the post and save it with the post. A dynamic block builds the content on the fly when rendered on the front end, see the [dynamic blocks guide](/docs/how-to-guides/block-tutorial/creating-dynamic-blocks.md).
```